src/ngraph/runtime/gpu/gpu_external_function.cpp

```diff
@@ -356,6 +361,59 @@ using namespace std;
     // to register cleanup handlers. We use it, and not atexit(), because
     // atexit() happens too late, when the JIT is no longer alive
     writer << "void *__dso_handle = 0;\n\n";
+    if (m_emit_timing)
+    {
+        writer << "// Declare debug timers\n";
+        vector<string> names;
+        size_t index = 0;
+        for (shared_ptr<Function> current_function : pass_manager.get_state().get_functions())
+        {
+            for (shared_ptr<Node> node : function_ordered_ops.at(current_function))
+            {
+                if (!node->is_parameter() && !node->is_constant())
+                {
+                    names.push_back(node->get_name());
+                    m_name_index_map.insert({node->get_name(), index++});
+                }
+            }
+        }
+        writer << "ngraph::stopwatch timers[" << names.size() << "];\n";
+        writer << "extern \"C\" size_t get_debug_timer_count() { return " << names.size()
+               << "; }\n";
+        writer << "extern \"C\" const char* get_debug_timer_name(size_t index)\n";
+        writer << "{\n";
```

Comment:
Use block begin/end thing here and following.

Reply:
fixed.